src/internal_rep.rs

```diff
@@ -248,6 +248,24 @@ impl Declared for TypeInfo {
 
 impl TypeInfo {
     pub fn new(td: TypeDecl, file: &SimpleFile<String, String>) -> Result<TypeInfo, CascadeErrors> {
+        let mut temp_vec = td.inherits.to_vec();
+        temp_vec.sort();
+        let mut iter = temp_vec.iter().peekable();
+        while let Some(cur_val) = iter.next() {
```

Comment:
I think the `let` followed by `while let` pattern here should be able to be written more cleanly as `for cur_val in temp_vec.iter().peekable() {`?

Reply:
We need the iterator object to be able to call peek on it.  Because of that I don't think we can do a for in loop here.

Reply:
Yeah, I guess you're right about that.  That's too bad.